pages/windows/if.md

```diff
@@ -0,0 +1,32 @@
+# if
+
+> Performs conditional processing in batch scripts.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Execute the specified commands if the condition is true:
+
+`if {{condition}} ({{commands}})`
+
+- Execute the specified commands if the condition is false:
+
+`if not {{condition}} ({{commands}})`
+
+- Execute the first specified commands if the condition is true otherwise execute the second specified commands:
+
+`if {{condition}} ({{first_commands}}) else ({{second_commands}})`
```

Comment:
Why we don't use the same command examples than the [common/if.md](https://github.com/tldr-pages/tldr/blob/main/pages/common/if.md) page? sounds easier to read and understand.
```suggestion
`if {{condition}} ({{echo "true"}}) else ({{echo "false"}})`
```

Reply:
Good suggestion. ;) I've replaced almost all occurrences of placeholders but preserved general descriptions.